### ISSUE TITLE
[bug]: "Module not found" errors running `cd sites/orgdocs ; yarn start`

### ISSUE THREAD

**Issue by BenJamesBen** [freesewing/freesewing#7206]
### What seems to be the problem? 🤔

4 compilation errors are seen when running `cd sites/orgdocs ; yarn start`:
```
Module not found: Error: Package path ./linedrawings is not exported from package .../node_modules/@freesewing/react-components (see exports field in .../node_modules/@freesewing/react-components/package.json)

--------------------------

Module not found: Error: Package path ./pattern is not exported from package .../node_modules/@freesewing/react-components (see exports field in .../node_modules/@freesewing/react-components/package.json)

--------------------------

Module not found: Error: Package path ./xray is not exported from package .../node_modules/@freesewing/react-components (see exports field in .../node_modules/@freesewing/react-components/package.json)

--------------------------

Module not found: Error: Package path ./pattern is not exported from package .../node_modules/@freesewing/react-components (see exports field in .../node_modules/@freesewing/react-components/package.json)
client (webpack 5.95.0) compiled with 4 errors
```

### Additional context

I suspect the problem is related to commit 76fbc5f's changes to `packages/react-components/package.json`.

I was able to work around the issue by re-adding the removed lines:

```diff
--- a/packages/react-components/package.json
+++ b/packages/react-components/package.json
@@ -23,7 +23,10 @@
     ".": {
       "internal": "./src/index.mjs",
       "default": "./dist/index.mjs"
-    }
+    },
+    "./linedrawings": "./src/linedrawings/index.mjs",
+    "./pattern": "./src/pattern/index.mjs",
+    "./xray": "./src/pattern-xray/index.mjs"
   },
   "scripts": {
     "build": "node build.mjs",
```

**Comment by BenJamesBen** [comment]
From Discord:
https://discord.com/channels/698854858052075530/944926520282054666/1301070420589547601
discord://discord.com/channels/698854858052075530/944926520282054666/1301070420589547601
> joostdecock — 10/29/2024 11:29 PM
That change to the package.json file of react-components was unintentional.
The file gets reconstructed by the reconfigure script, but I overlooked the change as it got snowed in since new contributors and updates were added which means a lot of files get changed.

The `package/react-components/package.json` file was reverted via #7208. However, running `yarn reconfigure` removes the lines from the file, causing the issue to be seen again.
